maintainers/scripts/rebuild-amount.sh

```diff
@@ -4,123 +4,130 @@ set -e
 # --print: avoid dependency on environment
 optPrint=
 if [ "$1" == "--print" ]; then
-	optPrint=true
-	shift
+    optPrint=true
+    shift
 fi
 
 if [ "$#" != 1 ] && [ "$#" != 2 ]; then
-	cat <<-EOF
-	Usage: $0 [--print] commit-spec [commit-spec]
-	    You need to be in a git-controlled nixpkgs tree.
-	    The current state of the tree will be used if the second commit is missing.
-	EOF
-	exit 1
+    cat <<EOF
+    Usage: $0 [--print] from-commit-spec [to-commit-spec]
+        You need to be in a git-controlled nixpkgs tree.
+        The current state of the tree will be used if the second commit is missing.
+
+        Examples:
+          effect of latest commit:
+              $ $0 HEAD^
+              $ $0 --print HEAD^
+          effect of the whole patch series for 'staging' branch:
+              $ $0 origin/staging staging
+EOF
+    exit 1
 fi
 
 # A slightly hacky way to get the config.
 parallel="$(echo 'config.rebuild-amount.parallel or false' | nix-repl . 2>/dev/null \
-			| grep -v '^\(nix-repl.*\)\?$' | tail -n 1 || true)"
+            | grep -v '^\(nix-repl.*\)\?$' | tail -n 1 || true)"
 
-echo "Estimating rebuild amount by counting changed Hydra jobs."
+echo "Estimating rebuild amount by counting changed Hydra jobs (parallel=$parallel)."
```

Comment:
```suggestion
echo "Estimating rebuild amount by counting changed Hydra jobs (parallel=${parallel:-unset})."
```
I think this is a bit more descriptive when parallel is not set in the config, than `(parallel=)`